ui/package.json

```diff
@@ -44,5 +46,8 @@
     "rimraf": "^3.0.2",
     "ts-jest": "^27.1.4",
```

Comment:
Can we remove ts-jest if we are using @swc/jest?

Reply:
what's the benefice of using `@swc/jest` instead of `ts-jest` ?

Reply:
The thought was it's faster than ts-jest but if I had tried to use it outside of components, @LukeTillman pointed out that it doesn't work with project references. Luke's recent PR also fixes issues I was running into so I'm going to remove the swc lines I added